### PR TITLE
[CALCITE-6131] There are duplicate sample tests in SqlTypeUtilTest

### PR DESCRIPTION
**Pull request by caicancai**
https://issues.apache.org/jira/browse/CALCITE-6131

### CODE REVIEW

**Review by mihaibudiu — APPROVED** [review]

File: core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java [hunk 1]
```diff
@@ -76,8 +76,6 @@ class SqlTypeUtilTest {
         is(true));
     assertThat(areSameFamily(ImmutableList.of(bigIntAndFloat, bigIntAndFloat)),
         is(true));
-    assertThat(areSameFamily(ImmutableList.of(bigIntAndFloat, bigIntAndFloat)),
```
Comment (JiajunBernoulli) [inline core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java:79]: It should be `assertThat(areSameFamily(ImmutableList.of(floatAndBigInt, bigIntAndFloat)),
        is(true));`

1. [bigIntAndFloat, floatAndBigInt]
2. [bigIntAndFloat, bigIntAndFloat]
3. [floatAndBigInt, bigIntAndFloat]
4. [floatAndBigInt, floatAndBigInt]

**Comment by caicancai** [comment]
cc @JiajunBernoulli

**Review by JiajunBernoulli — APPROVED** [review]

File: core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java (hunk 1 above)
Comment (chucheng92) [reply core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java:79]: +1 agree with JiaJun.

**Review by chucheng92 — APPROVED** [review]

**Comment by JiajunBernoulli** [comment]
@caicancai , The main branch is freezing because druid test failed.

I will merge it if druid test is fixed.

**Comment by caicancai** [comment]
> @caicancai , The main branch is freezing because druid test failed.
> 
> I will merge it if druid test is fixed.

OK, thank you for your reply

**Comment by caicancai** [comment]
> @caicancai CI passed. Please squash your commits.

Thank you,

> @caicancai CI passed. Please squash your commits.

It seems that there is something wrong with my operation. I will submit a PR again. Sorry.

**Comment by JiajunBernoulli** [comment]
@caicancai Not merge, we should rebase main branch.

**Comment by mihaibudiu** [comment]
Please follow the instructions here: https://calcite.apache.org/develop/ to combine all these 4 commits into a single one.
The commit message should be exactly the same as the title of this PR and the JIRA issue that you have filed.

**Comment by caicancai** [comment]
> @caicancai Not merge, we should rebase main branch.



> Please follow the instructions here: https://calcite.apache.org/develop/ to combine all these 4 commits into a single one. The commit message should be exactly the same as the title of this PR and the JIRA issue that you have filed.

Thanks for reminding me, I am trying, please forgive me for not being familiar with git rebase operation

**Comment by mihaibudiu** [comment]
Since you have changes to a single file, one simple approach is to create a new PR starting from the main branch.

- save the modified file to some external location: `core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java`
- check out the main branch (`git checkout main`)
- copy the modified file back over the main branch
- create a new branch `git checkout -b newbranch`
- add the changed file `git add core/`
- commit the changes `git commit -s -m"[CALCITE-6131] There are duplicate sample tests in SqlTypeUtilTest"`
- push the changes `git push origin newbranch`
- submit a new PR and close this one

**Comment by caicancai** [comment]
> As for rebase, usually it's enough to do `git fetch upstream; git rebase upstream main` But this can be more complicated if you have conflicts. In your case you have to not just rebase but combine all your 4 commits into one.

yes, I'm trying

**Pull request closed without merging**